pkg/cmd/controller/standalone_apiserver.go

```diff
@@ -58,7 +59,9 @@ func RunControllerServer(servingInfo configv1.HTTPServingInfo, kubeExternal clie
 	authz := newBypassAuthorizer(remoteAuthz, "/healthz", "/healthz/ready")
 	handler := apifilters.WithAuthorization(mux, authz, legacyscheme.Codecs)
 	// TODO need audiences
-	handler = apifilters.WithAuthentication(handler, authn, apifilters.Unauthorized(legacyscheme.Codecs, false), nil)
+
+	//handler = apifilters.WithAuthentication(handler, authn, apifilters.Unauthorized(legacyscheme.Codecs, false), nil)
+	handler = apifilters.WithAuthentication(handler, authn, apifilters.Unauthorized(serializer.WithoutConversionCodecFactory{CodecFactory: legacyscheme.Codecs}), nil)
```

Comment:
Outside the scope of this PR - do we need this? Do integration tests run a standalone apiserver any more?